src/templates/personas/persona_list.html

```diff
@@ -13,11 +13,11 @@
 {% endblock %}
 {% block content-header %}
     <div class="page-header">
-        <h1>{% trans "Members" %}({{ filter | length }})</h1>
+        <h1>{% trans "Members" %}({{ filter.qs | length }})</h1>
```

Comment:
> A count() call performs a SELECT COUNT(*) behind the scenes, so you should always use count() rather than loading all of the record into Python objects and calling len() on the result (unless you need to load the objects into memory anyway, in which case len() will be faster).

ってあるし、前確認したら `length` テンプレートフィルタは対象の `QuerySet` 
を Python オブジェクトにしてから `len()` 呼ぶ実装になってたから、今後気付いたところから `.count()` 使うように直していきたい

https://docs.djangoproject.com/en/1.10/ref/models/querysets/#django.db.models.query.QuerySet.count
https://github.com/kawazrepos/Kawaz3rd/pull/945/commits/5d0710d61d970f829dc6a1aa11cf66a041034b39